### ISSUE TITLE
Throttling

### ISSUE THREAD

**Issue by BrayBlade07** [dvx/lofi#196]
It always says throttling and i have to wait 20 seconds for it to stop to be able to use it

**Comment by stamoun** [comment]
Yeah, it's a known issue that is super high priority that I'll try and crack as soon as I'm done migrating Lofi to electron 17.

#107, #134, #177

Possible solution to implement #190
